### DISCUSSION TITLE
input() function in Jupyter Notebook on IDE Positron

### DISCUSSION THREAD

**Discussion by khavylo** [posit-dev/positron#4185] [Q&A] [answered]
How to get input in Jupyter Notebook on IDE Positron? I try to run input() function to get input from pop up textfield, but it goes in infinte loop.

**Comment by isabelizimm** [comment, marked as answer]
Ah you are correct 👀 this seems to be broken in notebooks. For context, if you run `input()` in a notebook in VSCode, there will be a popup at the top of the notebook to accept input. (Note `input()` does work in the console, if that is a helpful place to go for a workaround for now.)

<img width="1181" alt="Screenshot 2024-07-31 at 3 23 09 PM" src="https://github.com/user-attachments/assets/794536de-8071-489a-8558-d971c3845880">

In Positron, this does not appear, nor any other place to type in the `input()`.

<img width="1213" alt="Screenshot 2024-07-31 at 3 24 02 PM" src="https://github.com/user-attachments/assets/3568b763-a1fe-40a7-bcce-cedfbba12800">

I'm going to go ahead and promote this discussion as an issue so we can prioritize and get a fix in! Thank you so much for trying out Positron and reporting this!
